docs/install/Knative-with-MicroK8s.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Install on MicroK8s"
+linkTitle: "MicroK8s"
+weight: 10
+type: "docs"
+---
+
+[MicroK8s](https://microk8s.io) is a lightweight, powerful fully-conformant Kubernetes that tracks upstream releases and makes clustering trivial. It can run on any flavor of Linux that supports [Snap](https://snapcraft.io) packages. It can run on Windows and Mac OS using [Mutlipass](https://multipass.run).
+This guide walks you through the installation of Knative using MicroK8s.
+
+You can find [guides for other platforms here](./README.md).
+
+### Install MicroK8s
+
+```shell
+   sudo snap install --classic microk8s
+```
+
+## Alias MicroK8s kubectl for convenience
+
+```shell
+   sudo snap alias microk8s.kubectl kubectl
+```
+
+## Enable Knative
+
+```shell
+   echo 'N;' | microk8s.enable knative
+```
+This command will install all available Knative components.
+
+NOTE: As of this writing, MicroK8s comes with Knative version 0.7.1, this doc will be updated to reflect versioning changes. 
+
+You can check the status of Knative pods using the following commands:
+
+```shell
+   kubectl get pods -n knative-serving
+```
+
+```shell
+   kubectl get pods -n knative-eventing
+```
+
+```shell
+   kubectl get pods -n knative-monitoring
+```
+
+## Cleaning up
+
+Knative can be removed from MicroK8s using the following command:
+
+```shell
+sudo microk8s.disable knative
+```
+
+To delete MicroK8s along with Knative, Istio, and any deployed apps run:
+
+```shell
+sudo snap remove microk8s
+```
```

Comment:
One last request, is there a place people can go to ask for support for this process? A slack or IRC or email?

We do not have enough familiarity or expertise with MicroK8s to be able to help users going through this process that have problems.

Reply:
Absolutely! 

- This is the official Github repo for MicroK8s: https://github.com/ubuntu/microk8s
- The slack channel is in Kubernetes.slack.com #microk8s 
- There's also the Kubernetes discussion forum: https://discuss.kubernetes.io/tags/microk8s

Reply:
Should have been more clear. Can you add that information to the doc?

Reply:
ahh, thanks for clarifying. Done, added details at the beginning of the doc :)